pages/[lng]/register/index.tsx

```diff
@@ -1,5 +1,5 @@
 /* library package */
-import { FC, useState } from 'react'
+import { FC, useState, useRef, useEffect} from 'react'
```

Comment:
```suggestion
import { 
  FC,
  useState,
  useRef,
  useEffect
} from 'react'
```

legenda mengatakan jika improtnya lebih dari dua (atau 3) dibikin ke bawah.